s3.tf

```diff
@@ -1,7 +1,41 @@
 resource "aws_s3_bucket" "demo-example" {
```

Comment:
Accurics remediation:
Title: HIGH: S3 bucket should have versioning enabled
Description:Vulnerability: Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites/n/nRemediation:S3 bucket should have versioning enabled